packages/wallet/backend/src/account/service.ts

```diff
@@ -304,7 +309,9 @@ export class AccountService implements IAccountService {
     name = 'USD Account'
   ): Promise<Account | undefined> {
     const asset = (await this.rafikiClient.listAssets({ first: 100 })).find(
-      (asset) => asset.code === 'USD' && asset.scale === 2
+      (asset) =>
+        (asset.code === 'USD' && asset.scale === this.env.BASE_ASSET_SCALE) ||
+        undefined
```

Comment:
why `undefined` is used here?

Reply:
I agree, no need to have a manual fallback to undefined. `find` will fallback to `undefined` by itself if the item is not found.